integration/dev/dev_test.go

```diff
@@ -59,6 +59,11 @@ func TestDev(t *testing.T) {
 
 	eg := errgroup.Group{}
 	eg.Go(func() error {
+		_, err = helper.BuilderClient(t, ns.Name).ImageCreate(ctx, "test-app", "")
```

Comment:
again, not sure if we will do this permanently. you should keep all these ImageCreate adds in their own single commit that we can nuke later if we need to